RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerMultiFit.h

```diff
@@ -163,6 +164,9 @@ class EcalUncalibRecHitWorkerMultiFit final : public EcalUncalibRecHitWorkerBase
   bool kPoorRecoFlagEE_;
   double chi2ThreshEB_;
   double chi2ThreshEE_;
+
+  //Timing Cross Correlation Algo
+  EcalUncalibRecHitTimingCCAlgo computeCC_;
```

Comment:
perhaps a unique_ptr is a more practical choice since this is not needed for all variants of `EcalUncalibRecHitWorkerMultiFit`